src/connectivity-api/connectivity-qt/connectivityqt/connectivity.cpp

```diff
@@ -387,6 +390,34 @@ void Connectivity::setHotspotEnabled(bool enabled)
 
     // TODO Remove this when SDK switch widget isn't broken
     d->m_propertyCache->setProperty("HotspotEnabled", enabled);
+
+    // HACK: For switching MediaTek /dev/wmtWifi adapter mode between AP/STA
+    //       when hotspot enabled state is changed.
+    try
+    {
+        const char* wmtWifi_dev = "/dev/wmtWifi";
+
+        QFileInfo wmtWifi_fileinfo(wmtWifi_dev);
```

Comment:
Since you're already creating a QFIle below, please move that up here and use it's exists() function to avoid creating 2 variables that do the same thing.

https://doc.qt.io/qt-5/qfile.html#exists-1